bezier_library/doc/README.md

```diff
@@ -32,6 +32,14 @@ Input
 -----
 An input mesh (PLY file), a default mesh (PLY file) and several parameters for grinding (depth, diameter of the effector, percentage of covering, frequency of extrications and their depths).
 
+| Parameters | Description
+------------ | -----------
+`maximum_depth_of_pass_` | Maximum depth machining (in meters)
+`effector_diameter_` | Diameter of effector (in meters)
+`covering_percentage_` | Percentage of covering (decimal value)
+`extrication_coefficient_` | Extrication depth equal of the percentage of maximum_depth_of_path (coefficient)
```

Comment:
`maximum_depth_of_pass_`, use quotes so that it appears as a variable (not in plain text).
The underscore suffix is missing.